src/System.Buffers/ref/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netstandard;
       netstandard1.1;
+      uap10.0.15138;
```

Comment:
This is going to break in the uap vertical build unless you include the uap.depproj as part of that vertical build as well, or add a uap live build as well which is what I think I would do.